linker-interface/shared/src/main/scala/org/scalajs/linker/interface/ESFeatures.scala

```diff
@@ -26,36 +26,92 @@ import Fingerprint.FingerprintBuilder
  *    corresponding features if it supports them. Support for such options can
  *    be dropped in any subsequent version of the linker, including patch
  *    versions.
+ *  - Options whose name is of the form `avoidX` *hint* at the linker to avoid
+ *    the corresponding features *when it does not affect observable
+ *    semantics*. They are related to optimizations (for performance or code
+ *    size). The linker is free to ignore those options.
  */
 final class ESFeatures private (
-    /** Whether to use ECMAScript 2015 features, such as classes and arrow
-     *  functions.
-     */
-    val useECMAScript2015: Boolean,
-    /** EXPERIMENTAL: Whether to allow using ECMAScript `BigInt`s to implement
-     *  `Long`s.
-     */
-    val allowBigIntsForLongs: Boolean
+    // We define `val`s separately below so that we can attach Scaladoc to them
+    // (putting Scaladoc comments on constructor param `val`s has no effect)
```

Comment:
Wrong comment style? I thought multi-line comments need `/* ... */`.

Reply:
Yes, you're right. Fixed.